sips/sips/sip-009-nft-standard.md

```diff
@@ -71,6 +75,8 @@ The trait has three functions:
 |----------|-------|-------------| 
 |`transfer`|`{kind: "nft-transfer-failed", code: from-nft-transfer}`| Error if the call failed due to the underlying asset transfer. The code `from-nft-transfer` is the error code from the native asset transfer function|
 
+1. The methods `get-last-token-id`. `get-token-uri` and `get-owner` can be implemented as read-only functions.
```

Comment:
Should be a comma (not period) after `get-last-token-id`

Reply:
:+1: